### PR TITLE
Expand Solr docs to include expanded alternative

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

File: src/configuration/services/solr.md [hunk 1]
```diff
@@ -145,17 +147,49 @@ The relationships array would then look something like the following:
...
+                    configSet=mainconfig
+                    schema=arabic/schema.xml
+        endpoints:
+            na:
+                core: na_index
+            mideast:
+                core: mideast_index
+```
+
+In this example, the directory `.platform/configsets/solr6` contains the configuration definition for multiple cores.  There are then two cores created: `na_index` uses the defined configset, but specifically the `.platform/configsets/solr6/english/schema.xml` file, while `mideast_index` is identical except for using the `.platform/configsets/solr6/english/schema.xml` file.  Note that not all of Solrs core.properties file is available in the `core_properties` key.
```
Comment (sruggier) [inline src/configuration/services/solr.md]: There's a copy-paste mistake in the second schema.xml path here.

Comment (Crell) [inline src/configuration/services/solr.md]: Good catch, fixed.

File: src/configuration/services/solr.md [hunk 2]
```diff
@@ -145,17 +147,49 @@ The relationships array would then look something like the following:
...
+                    configSet=mainconfig
+                    schema=arabic/schema.xml
+        endpoints:
+            na:
+                core: na_index
+            mideast:
+                core: mideast_index
+```
+
+In this example, the directory `.platform/configsets/solr6` contains the configuration definition for multiple cores.  There are then two cores created: `na_index` uses the defined configset, but specifically the `.platform/configsets/solr6/english/schema.xml` file, while `mideast_index` is identical except for using the `.platform/configsets/solr6/arabic/schema.xml` file.  Note that not all of Solr's core.properties file is available in the `core_properties` key.
```
Comment (sruggier) [inline src/configuration/services/solr.md]: I think you mean to say something like:
> Note that not all core.properties features make sense to specify in the `core_properties`. Some keys, such as `name` and `dataDir`, are not supported, and may result in a solrconfig that fails to work as intended, or at all.

Comment (Crell) [inline src/configuration/services/solr.md]: Updated, using your language directly.

**Comment by sruggier** [comment]
Looks good to me :)

**Pull request merged**